.github/workflows/update_bcd-utils_api.yml

```diff
@@ -0,0 +1,23 @@
+name: Trigger mdn/bcd-utils API update
+
+on:
+  workflow_run:
+    workflows: ["Release package"]
+    types:
+      - completed
+
+jobs:
+  send_repo_dispatch_event:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.DISPATCH_PAT }}
+          script: |
+            const result = await github.rest.repos.createDispatchEvent({
+              owner: 'mdn',
+              repo: 'bcd-utils',
+              event_type: 'bcd_release',
+              client_payload: {}
```

Comment:
Do we need anything in the payload?

Reply:
The current implementation of the workflows [doesn't use](https://github.com/mdn/bcd-utils/blob/3a227173bf29ed8fd073f65ed8a482faf59e8b3c/.github/workflows/deploy-prod-api.yml#L6) the `workflow_trigger`s `notes` (optional) input; it's redundant. There is no point in sending payload that is not going to be utilized.